src/ucp/api/ucpx.h

```diff
@@ -19,8 +19,6 @@
 
 BEGIN_C_DECLS
 
-
```

Comment:
why

Reply:
The 'master' branch has this, I don't know why.